Monads/Monads/Failure.cs

```diff
@@ -73,8 +73,10 @@ public override Try<T> Filter(Predicate<T> predicate)
         {
             return this;
         }
- 
-
 
+        public override Try<T> Set(Action<T> setter)
```

Comment:
Hey Walter, I think the method name here shouldn't be Set(). I understand that you use it for setting values of properties, but it can be used for any action returning void, so it's intention is running an action. We should name it Run() at least, but I'm also considering just naming it Map() - and still return this in the Success class.